paddle/fluid/operators/matmul_op.cc

```diff
@@ -825,6 +825,22 @@ class MatMulOpGrad : public framework::OperatorWithKernel {
       context->SetOutputDim(y_grad_name, y_dims);
     }
   }
+
+  framework::OpKernelType GetExpectedKernelType(
+      const framework::ExecutionContext &ctx) const override {
+    auto input_data_type =
+        OperatorWithKernel::IndicateOrPromoteVarDataTypes(ctx, "X", "Y");
+
+#ifdef PADDLE_WITH_MKLDNN
+    using mkldnn::memory;
```

Comment:
using mkldnn::memory is not needed here

Reply:
Yes, sorry, I have copied that by an accident